src/backends/sentry_backend_inproc.c

```diff
@@ -168,8 +169,77 @@ shutdown_inproc_backend(sentry_backend_t *UNUSED(backend))
         SetUnhandledExceptionFilter(current_handler);
     }
 }
+
 #endif
 
+sentry_value_t
+sentry__registers_from_uctx(const sentry_ucontext_t *uctx)
+{
+    sentry_value_t registers = sentry_value_new_object();
+
+#ifdef SENTRY_PLATFORM_UNIX
+
+#elif defined(SENTRY_PLATFORM_WINDOWS)
+    PCONTEXT ctx = uctx->exception_ptrs.ContextRecord;
+
+#    define SET_REG(name, prop)                                                \
```

Comment:
Is this the same thing as `#define`?

Reply:
Not sure what you mean? Is it the formatting, or rather what this construct does?

Reply:
Yeah, the formatting was what I meant. I haven't seen that before.